ocsp/responder/redis/redis_source.go

```diff
@@ -128,17 +137,25 @@ func (src *redisSource) isStale(resp *ocsp.Response) bool {
 	return age > src.liveSigningPeriod
 }
 
-func (src *redisSource) signAndSave(ctx context.Context, req *ocsp.Request, cause string) (*responder.Response, error) {
+type signAndSaveCause string
+
+const (
+	causeStale    signAndSaveCause = "stale"
+	causeNotFound signAndSaveCause = "not_found"
```

Comment:
These two causes are never used; I think they're meant to be used on lines 123 and 109, respectively.